src/guides/hyperlinks.md

```diff
@@ -2,13 +2,13 @@
 A hyperlink is clickable-text. When the user clicks on the text, it directs them to a certain URL.
 
 ## General Hyperlinks
-You can use this method inside of `$description[]` and the `$addField[]` value field.
+You can use hyperlinks inside [`$description[]`](https://nilpointer-software.github.io/bdfd-wiki/bdscript/description.html), [`$addField[]`](https://nilpointer-software.github.io/bdfd-wiki/bdscript/addField.html) and in [webhook](https://nilpointer-software.github.io/bdfd-wiki/guides/webhooks.html) content/description.
```

Comment:
Use local hiperlinks. For example `[$description[]](../bdscript/description.md)`

Reply:
It's [relative links](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#relative-links)

Reply:
You can say that, but it won't work 100% of the time.
Let's say we have this code:
```
$description[[text](https://google.com/)]
```
You will just get this error: ❌ Function $description returned an error: No command body provided for $description!
But not if you do `\]`:
```
$description[[text\](https://google.com/)]
```
If you do `\]` like on BDS2- it works 100% of the time.

Your statement is correct, but I think it would be better to write in such a way that this is a way for BDS 2 and BDS Unstable.